src/announcements/tasks.py

```diff
@@ -35,12 +33,14 @@ def send_notifications(user_id, announcement_id):
 def get_users_to_email():
     students_to_email = list(
         CourseStudent.objects.all_users_for_active_semester()
+                             .filter(emailaddress__verified=True, emailaddress__primary=True)
```

Comment:
What's emailaddress__primary?

Reply:
django-allauth has a `EmailAddress` model which has a relationship to `User`. We don't want that feature where a user can have multiple email address so everytime they update their email via `ProfileForm`, there is a receiver `email_confirmed_handler` that calls `email_address.set_as_primary()` everytime an email is confirmed